### PR TITLE
Update readme with much more information

### PR DESCRIPTION
**Pull request by pixelmatrix**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Bitski JS SDK is a JavaScript library that connects your DApp with a user, a wal
 
 ### Getting Started
 
-In order to use Bitski you will need an account. You can set one up here: [https://www.bitski.com/account](https://www.bitski.com/account).
+In order to use Bitski you will need an account. You can set one up here: [https://www.bitski.com/signup](https://www.bitski.com/signup).
```
Comment (jtescher) [inline docs/README.md]: Did we want this to be develop.bitski.com/signup?

Comment (jtescher) [reply docs/README.md]: Or is that a subsequent PR once we have that in place

Comment (pat2man) [reply docs/README.md]: Yeah, we can link to that once it exists.

Comment (jtescher) [reply docs/README.md]: 👍

**Pull request merged**